grazer/src/main/webapp/WEB-INF/views/home.jsp

```diff
@@ -90,23 +90,14 @@ $(document).ready(function() {
 })
 
 </script>
-<script type="text/javascript">
-        $(function() {
-            var offset = $("#graph").offset();
-            var topPadding = 15;
-            $(window).scroll(function() {
-                if ($(window).scrollTop() > offset.top) {
-                    $("#graph").stop().animate({
-                        marginTop: $(window).scrollTop() - offset.top + topPadding
-                    });
-                } else {
-                    $("#graph").stop().animate({
-                        marginTop: 0
-                    });
-                };
-            });
-        });
-</script>
+
+<style>
+    .graph-outer { position: relative; }
+
```

Comment:
Do we need the style position:relative for .graph-outer. The position:relative tag is accompanied by either of these offset : top, bottom, right or left
https://www.w3schools.com/css/css_positioning.asp